staging/src/kubevirt.io/api/go.mod

```diff
@@ -21,8 +21,8 @@ require (
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/openshift/api v0.0.0-20230503133300-8bbcb7ca7183 // indirect
 	github.com/openshift/custom-resource-status v1.1.2 // indirect
-	golang.org/x/net v0.8.0 // indirect
-	golang.org/x/text v0.8.0 // indirect
+	golang.org/x/net v0.17.0 // indirect
```

Comment:
do you not need to put this in the `replace` section as well?
If not, then the go.sum can still hold old versions of golang.org/x/net

Reply:
https://stackoverflow.com/questions/69825533/why-does-go-sum-include-so-many-older-packages

Reply:
https://github.com/golang/go/wiki/Modules#is-gosum-a-lock-file-why-does-gosum-include-information-for-module-versions-i-am-no-longer-using